core/App/types/oca.ts

```diff
@@ -237,13 +250,29 @@ export class OCABundleResolver implements OCABundleResolverType {
     return Promise.resolve(new OCABundle(bundle, { ...this.options, language: language ?? this.options.language }))
   }
 
-  public resolve(credential: CredentialExchangeRecord, language = 'en'): Promise<OCABundle | undefined> {
+  public resolveDefaultBundle(credential: CredentialExchangeRecord, language = 'en'): Promise<OCABundle | undefined> {
     const credentialDefinitionId = credential.metadata.get(
       CredentialMetadataKeys.IndyCredential
     )?.credentialDefinitionId
     const schemaId = credential.metadata.get(CredentialMetadataKeys.IndyCredential)?.schemaId
+    return this.getDefaultBundle(credentialDefinitionId, schemaId, undefined, credential.connectionId, language)
```

Comment:
Once parameter lists start getting very long like this, I would consider passing an object instead, especially since many of these are optional